### ISSUE TITLE
Need a function that takes the product of a list of numbers

### ISSUE THREAD

**Issue by Juan-Pablo-Flores** [fickas/anl_test_repo#15]
This would be a great feature to have. If we can multiply a list of numbers instead of manually multiplying them by calculator, we can save a lot of time for other tasks!

**Comment by Juan-Pablo-Flores** [comment]
I agree. Surprised this has been overlooked. I will work on this issue.

**Comment by fickas** [comment]
Don't close until PR accepted.

**Comment by uomeercat** [comment]
This is another test from MeerCAT.

**Comment by fickas** [comment]
@fickas testing
